src/mocks/handlers.ts

```diff
@@ -0,0 +1,33 @@
+import { rest } from 'msw'
+import { charmander, pickachu } from './data'
+
+export const handlers = [
+  rest.get('https://api.pokemontcg.io/v2/cards', (req, res, ctx) => {
```

Comment:
I would explicitly type your request/response types for this (rest.get takes 2 generics, a request and response type)